package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@unocss/monorepo",
-  "version": "0.7.6",
+  "version": "0.7.7",
   "private": true,
   "scripts": {
     "bench": "npm -C bench run bench",
-    "build": "pnpm -r --parallel --filter !playground run build && pnpm -r run post",
+    "build": "pnpm -r --parallel --filter !playground run build && pnpm -C packages/cli run build && pnpm -r run post",
```

Comment:
should be:
`"build": "pnpm -r --parallel --filter !playground --filter !cli run build && pnpm -C cli run build && pnpm -r run post"`

you also need to run `update:test` script since the snapshots are changed: maybe will be better to merge main into your branch